src/client/config/local.config.json

```diff
@@ -1,4 +1,5 @@
 {
   "overwriteServerEndpoint": true,
-  "serverEndPointUrl": "localhost"
-}
\ No newline at end of file
+  "serverEndPointUrl": "localhost",
+  "serverPort": 8000
```

Comment:
What is this configuration ?

We have different way to access backend. If you develop without docker, if you develop with docker if you develop with minikube.
The production way is to access backend on port 80/443. It's also the way to access minikube (local kubernetes).

We need a hangout with front/backend/devops to me.